.buildkite/build-pipeline.yml

```diff
@@ -1,43 +1 @@
-# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
-
```

Comment:
we don't need Build and E2E CI jobs definition in 8.x branch as they align on main branch. I am not against for keeping them but it will be clean and clearly tell then we don't use 8.x definitions.